CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## Master
+
+##### Bug Fixes
+
+* Make linting faster than 0.5.0, but slower than 0.4.0  
+  [Norio Nomura](https://github.com/norio-nomura)
+  [#263](https://github.com/realm/SwiftLint/pull/263)
```

Comment:
Should I create an issue on SwiftLint or refer to https://github.com/jpsim/SourceKitten/issues/119 ?